driver/esp32/ili9XXX.py

```diff
@@ -542,12 +552,12 @@ def __init__(self,
         miso=5, mosi=18, clk=19, cs=13, dc=12, rst=4, power=14, backlight=15, backlight_on=0, power_on=0,
         spihost=esp.HSPI_HOST, spimode=0, mhz=40, factor=4, hybrid=True, width=240, height=320, start_x=0, start_y=0,
         colormode=COLOR_MODE_BGR, rot=PORTRAIT, invert=False, double_buffer=True, half_duplex=True,
-        asynchronous=False, initialize=True, color_format=lv.COLOR_FORMAT.NATIVE_REVERSED
+        asynchronous=False, initialize=True, color_format=lv.COLOR_FORMAT.NATIVE, swap_rgb565_bytes=True
     ):
 
         # Make sure Micropython was built such that color won't require processing before DMA
 
-        if lv.color_t.__SIZE__ != 2:
+        if lv.COLOR_DEPTH != 16:
```

Comment:
do not use this macro as it is no longer being used in LVGL. 

Use `lv.color_format_get_size(self.color_space)` instead

Reply:
True, now the display can have their own color format set in run time independently of `LV_COLOR_DEPTH`.

Reply:
Fixed.
Error messages are also updated.